config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -65,3 +65,28 @@ presubmits:
         - prow/cluster
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-test-infra-prow-checkconfig
+    branches:
+      - master
```

Comment:
nit: Should technically be `^master$` since this is a regexp. It might make more sense to omit the `branches` section entirely.